chassis/core/src/main/java/com/griddynamics/jagger/invoker/SimpleScenario.java

```diff
@@ -47,8 +47,9 @@ private SimpleScenario(Invoker<Q, R, E> invoker, Q query, E endpoint, SystemCloc
     }
 
     @Override
-    public void doTransaction() {
+    public boolean doTransaction() {
         invoker().invoke(query, endpoint);
+        return true;
```

Comment:
in this implementation we always return true - meaning always have non empty transactions